index.bs

```diff
@@ -2802,6 +2802,22 @@ which meant converting the names to camel-cased version.
 One such example is the <a href="https://w3c.github.io/image-resource/">image resource</a>.
 For this reason, if a key can clearly be expressed as a single word, that is recommended.
 
+<h3 id="serialization">Consider consumers when serializing</h3>
+Adding or extending features that involve a parser or a serializer should consider their effect on 
+serialization. The following are constituencies of serialization results that must be considered:
+
+ - Users - often reading result of serialization to validate expectations or learn
+ - Tools - use these results to enrich author experience, provide auditing, automation, testing etc.
```

Comment:
```suggestion
 - Tools - which may rely on serialization in many different ways
```

Reply:
feels like a couple of examples would be good